src/drc/datamodel/models.py

```diff
@@ -118,6 +122,8 @@ class InformatieObject(models.Model):
 
     objects = InformatieobjectQuerySet.as_manager()
 
+    IDENTIFICATIE_PREFIX = "DOCUMENT"
```

Comment:
Hoe wordt dit gebruikt? Ik zie in tests dat het werkt maar het model name werd toch overgenomen als prefix (wat door die slechte legacy resource naam niet lekker is).

Reply:
gaf problemen i.v.m. de max length - `ENKELVOUDIGINFORMATIEOBJECT-2019-0000000001` exceeds de 40 chars, en vond dit een meer leesbare oplossing. Nu wordt het dus `DOCUMENT-2019-0000000001`

Reply:
check.